example/ios/Flutter/Flutter.podspec

```diff
@@ -0,0 +1,18 @@
+#
+# NOTE: This podspec is NOT to be published. It is only used as a local source!
```

Comment:
As it says, this file is not supposed to be published

Reply:
Hi, I do not have a ios device to test the app in, please let me know if the PR is ready to be merged. Thanks for the contribution.

Reply:
@xni06 i deleted this file from PR